pages/views/index.11tydata.yml

```diff
@@ -0,0 +1,89 @@
+    title: Exadel Smart Library
+    slogan: UI component library with flair and precision
+    description: >
+      ESL is an open-source web components library that helps developers
+      create modern-looking, effective, and user-friendly interfaces faster.
+    ctas:
+      - text: Get started
+        url: /introduction
+        style: sec-blue
+      - text: See the library in action
+        url: /examples
+        style: sec-orange
+
+  benefits:
+    title: Why the ESL Web Components Library?
+    items:
+      - icon: benefits/efficient.svg
+        name: Efficient
+        text: >
+          Our team has reduced the work needed to create reusable UI components by giving you a reliable base
+          to customize and expand on for fast and accurate development.
+      - icon: benefits/fast.svg
+        name: Fast
+        text: >
+          With ESL, there’s no need to write lengthy markup — the reusable elements in our UI component library are small,
+          but they’re designed to be the optimal building blocks for your content.
+      - icon: benefits/light.svg
+        name: Light
+        text: >
+          We specially designed our reusable web components to work quickly and efficiently in browsers,
+          cutting out the unnecessary code that takes a toll on user experience.
+      - icon: benefits/powerful.svg
+        name: Powerful
+        text: >
+          While the modules in our web components library may be small, they’re also universal,
+          allowing you to mix and match the UI components at will to create large modules.
+      - icon: benefits/tested.svg
+        name: Tested
+        text: >
+          We cover all of our utilities with unit tests to make ESL the best front end component library,
+          which means every new addition works just as expected.
+      - icon: benefits/versatile.svg
+        name: Versatile
+        text: >
+          Our UI component library doesn't introduce any additional restrictions as it offers library
+          and framework-agnostic web components that work with any tech stack of your choice.
+
+  support:
+    title: Browser Support
+    text:
+      - >
+        Our web components library uses built-in browser functionality — Web API and ECMAScript 6 features
+        that are all fully supported by modern browsers such as Chrome, Firefox, Safari, or Edge.
+      - >
+        If you want our UI component library to work in older browsers,
+        you can use one of the polyfill lists — "light" or "full".
+    ctas:
+      - text: Read More
+        url: /introduction/browser-support
+        style: sec-blue
+
+  promo:
+    title: Let's Get Started
+    text: >
+      Join the array of consumers who have successfully utilized our UI component library.
+      ESL brings to you a number of ultra-light reusable UI components to create unique layouts.
+      The utilities to improve development experience are right within our library too.
```

Comment:
```suggestion
      ESL brings to you a number of ultra-light reusable UI components to create unique layouts, as well as utilities to improve the development experience.
```

Reply:
Thanks, applied with the latest corrections from Angela.